.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,3 +26,7 @@ Feel free to ping committers for the review!
 * [ ] If any new Jar binary package adding in your PR, please add License Notice according
   [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
 * [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
+* [ ] If you are contributing the connector code, please check that the following files are updated:
+  Connector description document and change log. https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2
+  Plugin-mapping. https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties
+  Seatunnel dist pom. https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml
```

Comment:
```suggestion
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it.
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
```

Reply:
Thank you for your review. I've fixed it.